404.html

```diff
@@ -24,6 +24,25 @@
       window.location.replace(newLoc);
     }
 
+		/**
+		* * Redirecting all newsroom pages and healthy thinking pages
+		*/
+		const windowHref = window.location.href;
+    const newsRegex = /newsroom\/(event|news|press-releases)/;
+    if (newsRegex.test(windowHref)) {
+			window.stop(); // Stop processing this page, make sure no 404 rendering is done
+      const newLoc = windowHref.replaceAll(/newsroom\/(event|news|press-releases)\//g, 'newsroom/');
+			window.location.replace(newLoc);
+    }
+
+    const healthifyThinkingRegex = /healthy-thinking\/(category)/;
+    if (healthifyThinkingRegex.test(windowHref)) {
+			window.stop(); // Stop processing this page, make sure no 404 rendering is done
+      const newLoc = windowHref.replaceAll(/healthy-thinking\/(category)\//g, 'healthy-thinking/');
+			console.log(newLoc);
```

Comment:
I assume `console.log` was just for your testing and can be removed.

Reply:
Yes, Thanks